shared/common-adapters/progress-indicator.android.js

```diff
@@ -6,7 +6,7 @@ export default class ProgressIndicator extends Component {
   render () {
     return <ProgressBarAndroid
       color={this.props.white ? globalColors.white : globalColors.black}
-      styleAttr='Small'
+      styleAttr={this.props.styleAttr || 'Small'}
       style={{...style, ...this.props.style}}/>
```

Comment:
> do we need to expose this as an external prop?

Getting styleAttr set to something else is the only way to get a larger spinner -- it looks like ActivityIndicatorIOS has small/large props, and ProgressBarAndroid has small/medium/large.

I'm not sure we'll ever want a "large" ProgressBarAndroid, it is gigantic.  Maybe we do:

`<ProgressIndicator size='Small'>` -> iOS small, Android small
`<ProgressIndicator size='Large'>` -> iOS large, Android medium

What do you think?